### PR TITLE
🌐 Add Japanese translation for `docs/ja/docs/advanced/templates.md`

### PR DESCRIPTION
**Pull request by souhub**
This PR translates advanced/templates.md in Japanese, help me review when you have time. Thank you!

### CODE REVIEW

File: docs/ja/docs/advanced/templates.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+* `Jinja2Templates`をインポートします。
+* あとで再利用可能な`templates`オブジェクトを作ります。
+* テンプレートを返す*パスオペレーション*内で`Request`パラメータを宣言します。
+* 作成した`templates`を使って`TemplateResponse`をレンダリングし、Jinja2の"context"のキーと値のペアの一つとして`request`を渡します。
+
+```Python hl_lines="4  11  15-16"
+{!../../../docs_src/templates/tutorial001.py!}
+```
+
+!!! note
```
Comment (tokusumi) [inline docs/ja/docs/advanced/templates.md:34]:
```suggestion
!!! note "備考"
```

File: docs/ja/docs/advanced/templates.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+`request`をJinja2のコンテキストのキーと値ののペアとして渡さなければいけないことに注意してください。そのため、*パスオペレーション*の中でもそれを宣言しなければなりません。
+
+!!! tip
```
Comment (tokusumi) [inline docs/ja/docs/advanced/templates.md:37]:
```suggestion
!!! tip "豆知識"
```

File: docs/ja/docs/advanced/templates.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+`response_class=HTMLResponse`を宣言することで、docs UIはレスポンスがHTML形式であると知ることができます。
+
+!!! note "Technical Details"
```
Comment (tokusumi) [inline docs/ja/docs/advanced/templates.md:40]:
```suggestion
!!! note "技術詳細"
```

File: docs/ja/docs/advanced/templates.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+また、`from starlette.templating import Jinja2Templates`も使用できます。
+
+    **FastAPI** provides the same `starlette.templating` as `fastapi.templating` just as a convenience for you, the developer. But most of the available responses come directly from Starlette. The same with `Request` and `StaticFiles`.
```
Comment (tokusumi) [inline docs/ja/docs/advanced/templates.md:43]:
```suggestion
    **FastAPI** は開発者の便利のために `starlette.templating` を `fastapi.templating` として提供します。しかし、利用可能なレスポンスのほとんどは `Request` や `StaticFiles` と同様にStarletteから直接渡されます。 
```

**Review by tokusumi — CHANGES_REQUESTED** [review]
Great work! I propose a little for not translated sentences remains.

File: docs/ja/docs/advanced/templates.md (hunk 4 above)
Comment (souhub) [reply docs/ja/docs/advanced/templates.md:43]: Thank you sooooooo much:) let me fix!

File: docs/ja/docs/advanced/templates.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+* あとで再利用可能な`templates`オブジェクトを作ります。
+* テンプレートを返す*パスオペレーション*内で`Request`パラメータを宣言します。
+* 作成した`templates`を使って`TemplateResponse`をレンダリングし、Jinja2の"context"のキーと値のペアの一つとして`request`を渡します。
+
+```Python hl_lines="4  11  15-16"
+{!../../../docs_src/templates/tutorial001.py!}
+```
+
+!!! note
+`request`をJinja2のコンテキストのキーと値ののペアとして渡さなければいけないことに注意してください。そのため、*パスオペレーション*の中でもそれを宣言しなければなりません。
```
Comment (komtaki) [inline docs/ja/docs/advanced/templates.md:35]:
```suggestion
    `request`をJinja2のコンテキストのキーと値ののペアとして渡さなければいけないことに注意してください。そのため、*パスオペレーション*の中でもそれを宣言しなければなりません。
```

A following sentence in a tips or note block needs indent like English. If no indent, cause broken css style and not was surrounded block.

https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/advanced/templates.md?plain=1#L35

File: docs/ja/docs/advanced/templates.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+!!! tip
+`response_class=HTMLResponse`を宣言することで、docs UIはレスポンスがHTML形式であると知ることができます。
```
Comment (komtaki) [inline docs/ja/docs/advanced/templates.md:38]:
```suggestion
    `response_class=HTMLResponse`を宣言することで、docs UIはレスポンスがHTML形式であると知ることができます。
```

Same reason.

File: docs/ja/docs/advanced/templates.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+!!! note "Technical Details"
+また、`from starlette.templating import Jinja2Templates`も使用できます。
```
Comment (komtaki) [inline docs/ja/docs/advanced/templates.md:41]:
```suggestion
    また、`from starlette.templating import Jinja2Templates`も使用できます。
```

Same reason.

**Review by komtaki — COMMENTED** [review]
Thanks you for your great work. I commented a few suggestion, so please check. 👍

**Comment by tiangolo** [comment]
📝 Docs preview for commit fd64969f51a262501568241dcb3716c274f14220 at: https://649a17df51d130136dc7df41--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**